File: 240802/T를 초과하는 연속 부분 수열/subsequence-above-t.py
n, t = map(int, input().split())
arr = list(map(int, input().split()))
ans = 0

for i in range(len(arr)):
    length = 0
    for j in range(i, len(arr)):
        if arr[j] > t:
            length += 1
        else:
            break
    ans = max(ans, length)
print(ans)

File: 240802/T를 초과하는 연속 부분 수열/subsequence-above-t.js
const fs = require("fs")
const input = fs.readFileSync(0).toString().trim().split("\n")
const [n, t] = input[0].split(" ").map(Number)
const nums = input[1].split(" ").map(Number)
let max = 0

for (let i = 0; i < n; i++){
    let len = 0
    for (let j = i; j < n; j++){
        if (nums[j] > t){
            len += 1
        } else {
            break
        }
    }
    max = Math.max(max, len)
}

console.log(max)